python/tictactoe.py

```diff
@@ -0,0 +1,68 @@
+import random
+from sys import setrecursionlimit
+
+username = input("What is your name?\n")
+timesTheLoopWent = 0
+whoWillBeFirst = random.randint(0, 1)
+symbolAssigner = random.randint(0, 1)
+global symbol
+symbol = ""
+global computerSymbol
+computerSymbol = ""
+global maxTimesTheLoopWent
+maxTimesTheLoopWent = 0
+cell = [0, 1, 2,
+        3, 4, 5,
+        6, 7, 8]
+setrecursionlimit(int(999999999))
```

Comment:
Messing with the stack call size is the only thing that is up to our expectations

Reply:
Oh

Reply:
Time to murder my computer in my testing then...
tbh idk what i am doing. pls help